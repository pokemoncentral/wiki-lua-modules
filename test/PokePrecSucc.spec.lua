-- "Test cases" for PokePrecSucc

local pps = require("PokePrecSucc")
local mw = require("mw")

-- Case tested: standard PokePrecSucc
print(pps.PokePrecSucc({ args = { " Staraptor " } }))

-- Case tested:
--      - automatic Pok√©mon name detection
--      - automatic subpage detection
mw.title.setTitle("Staraptor/Immagini")
print(pps.subpage({ args = {} }))
