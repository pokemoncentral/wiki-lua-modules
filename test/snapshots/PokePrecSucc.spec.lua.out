<div style="padding: 0.3em 0;">
<div class="flex flex-row flex-wrap flex-items-stretch flex-main-stretch roundy text-center black-text" style="padding: 0.2em; background-size: 100%; background-image: linear-gradient(to right, #9FA19F, #81B9EF);"><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[Staravia|&larr;]]&nbsp;[[File:Minim0397.png|Staravia|40px|link=Staravia]]&nbsp;[[Staravia|#0397: Staravia]]</div></div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-items-center flex-main-center roundy white-bg height-100" style="padding: 0.2em;">[[Elenco Pokémon secondo il Pokédex Nazionale|Staraptor]]</div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[File:Minim0399.png|Bidoof|40px|link=Bidoof]]&nbsp;[[Bidoof|#0399: Bidoof]]&nbsp;[[Bidoof|&rarr;]]</div></div></div></div></div>
<div style="padding: 0.3em 0;">
<div class="flex flex-row flex-wrap flex-items-stretch flex-main-stretch roundy text-center black-text" style="padding: 0.2em; background-size: 100%; background-image: linear-gradient(to right, #3FA129, #50413F);"><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[Codaurlante|&larr;]]&nbsp;[[File:Minim0985.png|Codaurlante|40px|link=Codaurlante]]&nbsp;[[Codaurlante|#0985: Codaurlante]]</div></div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-items-center flex-main-center roundy white-bg height-100" style="padding: 0.2em;">[[Elenco Pokémon secondo il Pokédex Nazionale|Fungofurioso]]</div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[File:Minim0987.png|Crinealato|40px|link=Crinealato]]&nbsp;[[Crinealato|#0987: Crinealato]]&nbsp;[[Crinealato|&rarr;]]</div></div></div></div></div>
<div style="padding: 0.3em 0;">
<div class="flex flex-row flex-wrap flex-items-stretch flex-main-stretch roundy text-center black-text" style="padding: 0.2em; background-size: 100%; background-image: linear-gradient(to right, #9FA19F, #81B9EF);"><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[Staravia/Immagini|&larr;]]&nbsp;[[File:Minim0397.png|Staravia|40px|link=Staravia]]&nbsp;[[Staravia/Immagini|#0397: Staravia]]</div></div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-items-center flex-main-center roundy white-bg height-100" style="padding: 0.2em;">[[Staraptor|Staraptor]]</div></div><div class="width-xl-30 width-sm-100" style="padding: 0.2em;">
<div class="flex flex-row flex-wrap flex-items-center flex-main-center roundy height-100 white-bg" style="padding: 0.2em;"><div class="width-xl-100">[[File:Minim0399.png|Bidoof|40px|link=Bidoof]]&nbsp;[[Bidoof/Immagini|#0399: Bidoof]]&nbsp;[[Bidoof/Immagini|&rarr;]]</div></div></div></div></div>
